client/src/connection/com/script.ts

```diff
@@ -0,0 +1,108 @@
+export const scriptContent = `
+class SASRunner{
+  [System.__ComObject] $objSAS
+  
+  [void]ResolveSystemVars(){
+  $code =
+@'
+   %let workDir = %sysfunc(pathname(work));
+   %put &=workDir;
+   %let rc = %sysfunc(dlgcdir("&workDir"));
+   run; 
+'@
+    $this.Run($code)
+    $varLogs = $this.FlushLog(4096)
```

Comment:
If a log is over 4MB, are we not going to get it all back?

Reply:
Oh, nevermind. It looks like that is just for the code immediately above it, which I assume you (and I agree) are confident will always result in a log less than 4MB.

Reply:
Exactly right. That log output size should never reach the chunksize.